main.go

```diff
@@ -41,14 +42,21 @@ func main() {
 			fmt.Printf("Error: unable to parse repository name %s.\n", entry)
 			os.Exit(1)
 		}
-
+		if v.fork == "true" {
+		// start a new  build
+		_, err = client.BuildFork(owner, name, build.Number)
+			if err != nil {
+				fmt.Printf("Error: unable to trigger a new build for %s.\n", entry)
+				os.Exit(1)
+		}
+		} else {
 		// get the latest build for the specified repository
 		build, err := client.BuildLast(owner, name, branch)
 		if err != nil {
```

Comment:
this needs to happen before `client.BuildFork` so we know which build number to fork from